src/Management/ResourceBuilder.php

```diff
@@ -410,6 +412,14 @@ private function buildUser(array $data): User
         ]);
     }
 
+    private function buildOrganization(array $data): Organization
+    {
+        return $this->createObject(Organization::class, [
+            'name' => $data['name'],
+            'sys' => $this->buildSystemProperties($data['sys'])
```

Comment:
Oh how I hate these gigantic "we just changed the styleguide" patches. I always do them one rule at the time :)